generator/templates/rust/src/lib.rs.mustache

```diff
@@ -13,6 +13,9 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 }}
+{{! TODO(#120) - re-enable dead_code warnings once OpenAPI generates some functions }}
+#![allow(dead_code)]
```

Comment:
nit: I think this may likely always be needed unless we write tests for every generated method, right?

Reply:
No, once the define some public methods that use all the private stuff we will be fine.  The issue here is that the OpenAPI translator does not create any public APIs yet.